taxonomy-editor-frontend/src/pages/project/editentry/ListEntryChildren.tsx

```diff
@@ -122,6 +128,14 @@ const ListEntryChildren = ({ url, urlPrefix, setUpdateNodeChildren }) => {
         </IconButton>
       </Stack>
 
+      {/* Renders warning message to save changes to be able to click on a child node */}
+      {hasChanges ? (
```

Comment:
nit: Since we don't show anything if there are no changes, I would rather write `hasChanges &&`